tests/integration/components/x-toggle/component-test.js

```diff
@@ -0,0 +1,97 @@
+import { moduleForComponent, test } from 'ember-qunit';
```

Comment:
On a related note, does setting the onValue property to "False:false" result in a boolean value or a string value?